js/src/features/project/components/Team/TeamAddMemberForm.tsx

```diff
@@ -0,0 +1,110 @@
+import React, { useCallback, useMemo, useState } from "react";
+import { Button, Form, Select } from "antd";
+
+import { AddProjectTeamMemberRequest, Role } from "ProjectModels";
+import { EmployeeShort } from "HumanResourceModels";
+
+interface Props {
+  teamId: string;
+  roles: Role[];
+  add: (form: AddProjectTeamMemberRequest) => void;
+  employees: EmployeeShort[];
+}
+
+const { Option } = Select;
+
+export const TeamAddMemberForm: React.FC<Props> = ({
+  teamId,
+  roles,
+  add,
+  employees,
+}) => {
+  const [form] = Form.useForm();
+  const [searchResult, setSearchResult] = useState<
+    { label: string; value: string }[]
+  >([]);
+
+  const users = useMemo(
```

Comment:
May be you can try to use selector for this autocomplete.

Reply:
fix